### DISCUSSION TITLE
Which is good isar / sqlite engine?

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [isar/isar#1361] [Help] [answered]
i want to use large amount of data maybe more than 10gb / more than 100gb, which one is good for long term?

**Comment by simc** [comment]
I strongly recommend not to store 10GB in a database on an embedded device. If we talk about desktop the Isar engine can easily handle 10GB

**Reply by ClarissaGazalaEvanthe** [reply]
i will use database more than 100gb on linux server, which one is good isar/sqlite?

**Reply by simc** [reply, marked as answer]
Isar engine is the right choice for that purpose
